### PR TITLE
Pull Repos on Signup

### PR DESCRIPTION
**Pull request by samsheff**

### CODE REVIEW

**Comment by samsheff** [comment]
@aaronmallen Give this a look over and let me know what you think

**Comment by aaronmallen** [comment]
@samsheff let's use shoulda-matchers for validation specs,  they're better for readability.

**Comment by samsheff** [comment]
@aaronmallen Agreed. Fixing

File: spec/models/repo_spec.rb [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
-  pending("Not Yet Implemented")
+
+  context 'with valid attributes' do
+    let!(:repo) { create :repo }
+
+    it 'should belong to a user' do
+      expect(repo.user).to be_a(User)
+    end 
+
+    it 'should have a name' do
```
Comment (aaronmallen) [inline spec/models/repo_spec.rb]: Swap this out with a shoulda-matcher

File: spec/models/repo_spec.rb [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+      expect(repo.name).to eq("Test Repo")      
+    end
+
+    it 'should require a name' do
```
Comment (aaronmallen) [inline spec/models/repo_spec.rb]: Swap this out with a shoulda-matcher

File: spec/models/repo_spec.rb [hunk 3]
```diff
@@ -1,5 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe Repo, type: :model do
-  pending("Not Yet Implemented")
+
+  context 'with valid attributes' do
+    let!(:repo) { create :repo }
+
+    it 'should belong to a user' do
```
Comment (aaronmallen) [inline spec/models/repo_spec.rb]: Swap this out with a shoulda-matcher

**Comment by samsheff** [comment]
@aaronmallen Done, pushing that now

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -36,7 +36,26 @@
       new_user = create(:user)
       expect(new_user.respond_to? :github_token)
     end
+ 
+    it 'should have many repos' do
```
Comment (aaronmallen) [inline spec/models/user_spec.rb]: Swap this out with a shoulda-matcher

**Comment by samsheff** [comment]
@aaronmallen Updated!

**Pull request merged**